fjord/base/browsers.py

```diff
@@ -98,11 +98,14 @@ def parse_ua(ua):
             browser_version = part[1]
         elif 'Iceweasel' in part:
             browser = 'Iceweasel'
+        elif 'FxiOS' in part:
+            browser = 'Firefox for iOS'
+            browser_version = part[1]
 
     platform = platform_parts.pop(0)
     platform_version = UNKNOWN
 
-    while platform in ['X11', 'Ubuntu', 'U']:
+    while platform in ['X11', 'Ubuntu', 'U', 'iPad', 'iPhone']:
```

Comment:
I imagine that you'll want to know if the user is on an iPhone or an iPad at some point, since FxiOS runs on both and has a fairly distinct UI (afaik). I'm not sure if the data model here supports that though, unfortunately. Maybe keep this in mind when you are doing your big rewrite?